dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPQProp.mo

```diff
@@ -0,0 +1,63 @@
+within Dynawo.Electrical.Machines.SignalN;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model GeneratorPQProp "Model for generator PQ based on SignalN for the frequency handling and with a proportional reactive power regulation"
+
+ import Dynawo.Electrical.Machines;
+ import Dynawo.AdditionalIcons;
+ import Dynawo.Connectors;
+
+  extends Machines.BaseClasses.BaseGeneratorSimplified;
```

Comment:
Could you extend Machines.SignalN.BaseClasses.BaseGeneratorSignalN?

Reply:
I could yes. I will modify BaseGeneratorSignalN to have only the things that are common to all generators.